api/tests/routes/pro/delete_collective_offers_template_test.py

```diff
@@ -0,0 +1,121 @@
+UPLOAD_FOLDER = settings.LOCAL_STORAGE_DIR / CollectiveOfferTemplate.FOLDER
+
+
+@pytest.mark.usefixtures("db_session")
+class DeleteImageFromFileTest:
+    def test_delete_from_file(self, client):
+        offer = CollectiveOfferTemplateFactory(
+            mentalDisabilityCompliant=False,
+            contactEmail="johndoe@yopmail.com",
+            contactPhone="0600000000",
+            subcategoryId="CINE_PLEIN_AIR",
+            domains=[],
+        )
+
+        offerers_factories.UserOffererFactory(
+            user__email="user@example.com",
+            offerer=offer.venue.managingOfferer,
+        )
+
+        # given
+        thumb = (IMAGES_DIR / "mouette_full_size.jpg").read_bytes()
+        data = {
+            "credit": "John Do",
+            "thumb": (BytesIO(thumb), "image.jpg"),
+            "croppingRectX": 0.0,
+            "croppingRectY": 0.0,
+            "croppingRectHeight": 0.9,
+            "croppingRectWidth": 0.6,
+        }
+
+        response = client.with_session_auth(email="user@example.com").patch(
+            f"/collective/offers-template/{humanize(offer.id)}/image", form=data
+        )
+
+        # when
+
+        response = client.with_session_auth(email="user@example.com").delete(
+            f"/collective/offers-template/{humanize(offer.id)}/image"
+        )
+
+        # then
+
+        assert response.status_code == 204
+        assert (UPLOAD_FOLDER / offer._get_image_storage_id()).exists() == False
+
+    def test_forbidden_offer(self, client):
+        # given
+        offer = CollectiveOfferTemplateFactory(
+            mentalDisabilityCompliant=False,
+            contactEmail="johndoe@yopmail.com",
+            contactPhone="0600000000",
+            subcategoryId="CINE_PLEIN_AIR",
+            domains=[],
+        )
+
+        offer2 = CollectiveOfferTemplateFactory(
+            mentalDisabilityCompliant=False,
+            contactEmail="johndoe@yopmail.com",
+            contactPhone="0600000000",
+            subcategoryId="CINE_PLEIN_AIR",
+            domains=[],
+        )
+
+        offerers_factories.UserOffererFactory(
+            user__email="user@example.com",
+            offerer=offer.venue.managingOfferer,
+        )
+
+        # when
+        response = client.with_session_auth(email="user@example.com").delete(
+            f"/collective/offers-template/{humanize(offer2.id)}/image"
+        )
+
+        # then
+        assert response.status_code == 403
+
+    def test_not_authentified(self, client):
+        # given
+        thumb = (IMAGES_DIR / "mouette_full_size.jpg").read_bytes()
+        data = {
+            "credit": "John Do",
+            "thumb": (BytesIO(thumb), "image.jpg"),
+            "croppingRectX": 0.0,
+            "croppingRectY": 0.0,
+            "croppingRectHeight": 0.9,
+            "croppingRectWidth": 0.6,
+        }
+
+        # when
+        response = client.patch("/collective/offers-template/0/image", form=data)
```

Comment:
(détail) On n'a pas besoin d'une vraie image (donc `thumb = b"dummy"` devrait être suffisant). Et à mon avis, on n'a même pas besoin de fournir un payload valide (puisque l'authentification survient avant la validation du payload). On pourrait donc simplement écrire : 

```suggestion
        response = client.patch("/collective/offers-template/0/image")
```

Idem dans le test de suppression.